doc/tutorials/using_zephyr_as_uos.rst

```diff
@@ -92,11 +92,10 @@ Steps for Using Zephyr as User VM
    the ACRN Service VM, then you will need to transfer this image to the
    ACRN Service VM (via, e.g, a USB drive or network)
 
-#. Follow XXX to boot "The ACRN Service OS" based on Clear Linux OS 28620
-   (ACRN tag: acrn-2019w14.3-140000p)
+#. Follow `Build and Install ACRN on Ubuntu
+   <https://projectacrn.github.io/latest/getting-started/rt_industry_ubuntu.html#build-and-install-acrn-on-ubuntu>`_
+   to boot "The ACRN Service OS" based on Ubnuntu OS(ACRN tag: acrn-2019w14.3-140000p)
```

Comment:
Please update the ACRN version, assuming a newer one was tested. That one is over 1,5 years old.

Reply:
Done!